src/reducers/index.js

```diff
@@ -3,8 +3,8 @@
 import { combineReducers } from 'redux'
 import authenticate from './authenticate'
 import {
-  removeMyItem, setItemsOrSelections, setBaseURL, showResourceURIMessage, setMyItemsLang,
-  showGroupChooser, closeGroupChooser, showRdfPreview,
+  removeAllContent, removeMyItem, setItemsOrSelections, setBaseURL, showResourceURIMessage, setMyItemsLang,
```

Comment:
Why are you adding removeAllContent back?

Reply:
Cruft, that shouldn't have stayed there. Will fix.